app/views/admin/councillor_contributions/_details.html.arb

```diff
@@ -0,0 +1,13 @@
+table do
+  tr do
+    th "Contributor Name"
+    th "Contributor Email"
+    th "Authority"
+  end
+  tr do
+    td a contributor.name, href:admin_contributor_path
```

Comment:
I deleted the link as it is not needed. It was going to contributor detail show page.